tools/schema_tools/prep_opsim.py

```diff
@@ -101,67 +138,214 @@ def offsetHex():
             vertex_count += 1
     return offsets
 
-def add_dither(database, simname, overwrite=True):
-    ## Add the Krughoff-Jones dithering pattern to the opsim output
-    ## adds three columns (ditheredRA, ditheredDec, and vertex) and indexes
-    #connect to the database
+def offsetRandom(noffsets, inHex=True):
+    """ 
+    Returns translational random dither offsets in radians.
+    
+    Required input
+    --------------
+    * noffsets: int: number of offsets needed.
+    
+    Optional input
+    --------------
+    * inHex: bool: set to False if dont want to restrict the dithers to within
+                   the hexagon inscribing the FOV. Default: True
+                  
+    """
+    import numpy as np
+    from lsst.sims.maf.stackers import inHexagon
```

Comment:
Note, a dependence on `sims_utils` is going to bring with it a dependence on `boost` because `boost` is used by the LSST unit testing framework.  Compiling `boost` can also be time-consuming (though not as much so as `afw`).  We could talk about stripping down `sims_utils` so that it uses a more bare-bones unit testing framework, rather than the official LSST framework, if need be.